<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="localWorkspace">
         <attribute name="title">
          <string>Local Workspace</string>
         </attribute>
         <widget class="QSplitter" name="changedFilesSplitter">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>181</width>
            <height>141</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>141</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListView" name="changedFileslistView"/>
          <widget class="QLabel" name="changedFilesLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Changed Files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="savedFilesSplitter">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>20</y>
            <width>181</width>
            <height>151</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListView" name="stagedFilesListView">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
          <widget class="QLabel" name="filesToSaveLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Files to Save&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="ignoredFilesSplitter">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>230</y>
            <width>181</width>
            <height>141</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListView" name="ignoredFilesListView"/>
          <widget class="QLabel" name="filesToIgnoreButton">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Files to Ignore&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="gitIgnoreButton">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>160</y>
            <width>91</width>
            <height>201</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/downArrow.png</normaloff>:/images/downArrow.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>90</width>
            <height>200</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="gitAddButton">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>20</y>
            <width>91</width>
            <height>201</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/images/topArrow15.png</normaloff>
            <normalon>:/images/topArrow15.png</normalon>
            <disabledoff>:/images/topArrow15.png</disabledoff>
            <disabledon>:/images/topArrow15.png</disabledon>
            <activeoff>:/images/topArrow15.png</activeoff>
            <activeon>:/images/topArrow15.png</activeon>
            <selectedoff>:/images/topArrow15.png</selectedoff>
            <selectedon>:/images/topArrow15.png</selectedon>:/images/topArrow15.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>90</width>
            <height>200</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="reload">
          <property name="geometry">
           <rect>
            <x>680</x>
            <y>410</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_3">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>20</y>
            <width>281</width>
            <height>351</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>261</width>
             <height>331</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="shipButton">
              <property name="text">
               <string>Ship</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="withLabel">
              <property name="text">
               <string>with title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="commitName"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="andLabel">
              <property name="text">
               <string>and description</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QTextEdit" name="commitDescription"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="packageTree">
         <attribute name="title">
          <string>Package Tree</string>
         </attribute>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>40</y>
            <width>211</width>
            <height>191</height>
           </rect>
          </property>
          <property name="title">
           <string>Branches</string>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>191</width>
             <height>159</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="mergeBranch">
            <property name="text">
             <string>Merge Branches</string>
            </property>
           </widget>
           <widget class="QPushButton" name="createNewBranch">
            <property name="text">
             <string>Create New Branch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="deleteBranch">
            <property name="text">
             <string>Delete Branch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="renameBranch">
            <property name="text">
             <string>Rename Branch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="changeCurrentBranch">
            <property name="text">
             <string>Change Current Branch</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>540</x>
            <y>260</y>
            <width>211</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title">
           <string>Remote Repository</string>
          </property>
          <widget class="QSplitter" name="splitter_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>19</y>
             <width>191</width>
             <height>101</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QPushButton" name="syncToButton">
            <property name="text">
             <string>Sync to Remote Branch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="syncFromButton">
            <property name="text">
             <string>Sync From Remote Branch</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Refresh Remote Branches</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QPlainTextEdit" name="gitLogTextEdit">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>481</width>
            <height>391</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionConfigure"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUser_s_Manual"/>
   </widget>
   <widget class="QMenu" name="menuWizards">
    <property name="title">
     <string>Wizards</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionExisting_Project"/>
    <addaction name="actionRemote_Repository_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWizards"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUser_s_Manual">
   <property name="text">
    <string>User's Manual</string>
   </property>
  </action>
  <action name="actionExisting_Project">
   <property name="text">
    <string>Existing Project</string>
   </property>
  </action>
  <action name="actionShare_Project">
   <property name="text">
    <string>Share Project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionRemote_Repository">
   <property name="text">
    <string>Remote Repository</string>
   </property>
  </action>
  <action name="actionRemote_Repository_2">
   <property name="text">
    <string>Remote Repository</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>changedFileslistView</tabstop>
  <tabstop>stagedFilesListView</tabstop>
  <tabstop>ignoredFilesListView</tabstop>
  <tabstop>commitName</tabstop>
  <tabstop>commitDescription</tabstop>
  <tabstop>mergeBranch</tabstop>
  <tabstop>createNewBranch</tabstop>
  <tabstop>deleteBranch</tabstop>
  <tabstop>renameBranch</tabstop>
  <tabstop>changeCurrentBranch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
